src/core/Akka.Persistence/persistence.conf

```diff
@@ -171,6 +174,9 @@ akka.persistence {
         # Dispatcher for the plugin actor.
         plugin-dispatcher = "akka.persistence.dispatchers.default-plugin-dispatcher"
 
+        # Default serializer used as manifest serializer when applicable and payload serializer when no specific binding overrides are specified
+        serializer = "json"
+
```

Comment:
Same goes for this one, only then defined in the ` snapshot-store-plugin-fallback`